control-plane/pkg/reconciler/trigger/v2/triggerv2.go

```diff
@@ -41,22 +43,42 @@ import (
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/config"
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/kafka"
 	kafkalogging "knative.dev/eventing-kafka-broker/control-plane/pkg/logging"
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/base"
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/security"
+
+	kedaclientset "knative.dev/eventing-autoscaler-keda/third_party/pkg/client/clientset/versioned"
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/apis/eventing/v1alpha1"
+
+	kedafunc "knative.dev/eventing-kafka-broker/control-plane/pkg/keda"
 )
 
 const (
 	deliveryOrderAnnotation = "kafka.eventing.knative.dev/delivery.order"
+	// TopicPrefix is the Kafka Broker topic prefix - (topic name: knative-broker-<broker-namespace>-<broker-name>).
+	TopicPrefix = "knative-broker-"
+
+	// ExternalTopicAnnotation for using external kafka topic for the broker
+	ExternalTopicAnnotation = "kafka.eventing.knative.dev/external.topic"
 )
 
 type Reconciler struct {
+	*base.Reconciler
```

Comment:
Why do we need it?

Reply:
It was for the [secretProviderFunc](https://github.com/knative-sandbox/eventing-kafka-broker/pull/2595/files#diff-17a28f62dd8ddd363016236484edbadd40e76c69068b777b11030d93e0c94578R146) but redid that one..